compiler/bindings/c/iree/compiler/embedding_api.h

```diff
@@ -248,6 +248,7 @@ ireeCompilerInvocationSetVerifyIR(iree_compiler_invocation_t *inv, bool enable);
 enum iree_compiler_pipeline_t {
```

Comment:
Minimally, can you please add some documentation to each of these. We should have had a blurb the whole time but it was easier to ignore when there was just one... then there was two, and now three.